ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClient.java

```diff
@@ -226,19 +228,17 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
         }
     }
 
-    private long estimateFileRawSize(String filePath, IngestionProperties.DataFormat format) {
+    private long estimateFileRawSize(String filePath, boolean isCompressible) {
```

Comment:
I don't think we have this logic in the other SDKs. We probably want it, right? I'll add it to the list